tempto-core/src/main/java/io/trino/tempto/sql/SqlContexts.java

```diff
@@ -34,11 +34,17 @@
      *
      * @param selectSql sql select statement used to create view
      * @return viewContextProvider
+     * @deprecated This method uses internally the default query executor which is deprecated. Use {@link #createViewAs(String, QueryExecutor)} instead.
```

Comment:
Also add `@Deprecated` annotation.

Reply:
Thanks for pointing this out. I did just add `@deprecated` in the javadoc before.